ng/src/gmp/commands/entities.js

```diff
@@ -145,6 +145,18 @@ class EntitiesCommand extends HttpCommand {
       return this.delete(entities, extra_params);
     }).then(response => response.setData(deleted));
   }
+
+  transformAggregates(response) {
+    return response.setData(
+      response.data.get_aggregate.get_aggregates_response.aggregate);
+  }
+
+  getAggegates(params = {}) {
```

Comment:
Please fix typo to getAggregates(). This function is called with the wrong name as well. I'll mark those cases...